keras_cv/layers/transformer_encoder.py

```diff
@@ -23,7 +23,6 @@ class TransformerEncoder(layers.Layer):
 
     Args:
         project_dim: the dimensionality of the projection of the encoder
```

Comment:
add statement that this is the output of multihead

Reply:
Sure!

Reply:
Done